app/assets/javascripts/past_teams.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  var $selectedYear = $('select#past_teams');
+
+  $selectedYear.on('change', function(){
+    var yearParam = "?year="+this.value;
+    var newUrl;
+    var removed = window.location.search;
+
+    if(removed.length>0){
```

Comment:
I cannot wrap my head around how that would behave in the browser I'm afraid. Can someone else check that pls? 😁